README.md

```diff
@@ -50,20 +70,4 @@ Subscribestar is our other subscription-based donation platform. Has fewer featu
 
 [<img src="Documents/Images/crypto.png" alt="Crypto" width="150" align="left">](SpaceStation3D.crypto)
```

Comment:
This link doesn't work. Redirects to github 404.

Reply:
'SpaceStation3d.crypto' doesn't work as a real link unless we host the domain, which isn't necessary for us. We use it as a redirect for the 4 cryptos we accept so people don't need to send directly to a hash address (and it doesn't need to be hosted for this).

I will just remove the link part of this to reduce confusion.